tests/Unit/Numerical/LinearAlgebra/Test_InvertMatrix.cpp

```diff
@@ -0,0 +1,172 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <catch.hpp>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Numerical/LinearAlgebra/InvertMatrix.hpp"
+#include "Utilities/TypeTraits.hpp"
+
+namespace {
+using generic = tmpl::integral_list<int32_t, 2, 1>;
+using symmetric = tmpl::integral_list<int32_t, 1, 1>;
+
+template <size_t Dim>
+using matrix_ii = Tensor<double, symmetric,
```

Comment:
I would like to avoid the word matrix as Matrix is used for run-time matrices in SpECTRE and Tensors are compile-time and limited to 4-dimensions